ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -227,13 +238,26 @@ - (void)webView:(__unused WKWebView *)webView decidePolicyForNavigationAction:(W
   NSString* scheme = url.scheme;
   
   BOOL isJSNavigation = [scheme isEqualToString:RCTJSNavigationScheme];
-  
+
+  static NSDictionary<NSNumber *, NSString *> *navigationTypes;
```

Comment:
If return a hardcoded number, we have to read the Objective-C document to find out that number represents.

But this change is breaking, so it may be inappropiriate.